notebooks/Readme.md

```diff
@@ -0,0 +1,27 @@
+## Notebook Examples
+
+In this folder you can find [jupyter notebook](https://jupyter.org/) examples to illustrate how to automate certain aspects of, and around ilastik from Python.
+
+The structure in each folder will follow the same scheme:
+
+* notebooks (here)
+  * `subfolder` for a certain task
+    * `environment.yml`: complete environment to use with conda, includes all packages necessary to run
+    * `task.ipynb`: Notebook that illustrate how to perform the task
+
+We use _conda_ to develop Python and recommend it for scientific Python development.
+It is assumed that you have already installed it and are familiar with using a Terminal.
+
+In order to run the notebook type the following in a Terminal/Command Line:
+
+```bash
+$ conda env create -f environment.yml
+# this will produce a lot of output
+
+$ conda activate <created_environment_name>
```

Comment:
won't the name always be h5convert, as defined in the `environment.yml` file?

Reply:
right :) thx for that one

Reply:
ah, wait. This is the top level directory. So readme here should be generic :)